docs/proposal-config-changes.md

```diff
@@ -225,10 +245,10 @@ SPEC = {
     'monitor': {
         'sort order': [VDR.V_STRING, 'definition', 'alphanumeric'],
     },
-    'job platforms': {
+    'platforms': {
         '__MANY__': {
-            'batch system': [VDR.V_STRING, 'background'],
-            'batch submit command template': [VDR.V_STRING],
+            'job runner': [VDR.V_STRING, 'background'],
```

Comment:
Quick question, do we want to refer to the default "job runner" as "background" or None?

Reply:
Would `None` mean you can't run jobs (i.e. users have to set an explicit value or their jobs won't run), or would it mean "background"?   I can see an argument both ways, but having to be explicit would making writing dummy test suites a little bit more difficult.

Reply:
I was thinking that `None` would mean `background`.

Reply:
`None` could also be better on grounds that "background" is not strictly correct (in the shell "background process" sense - done with `&` and `wait`able).   It more (as you say) that we run the job directly without submitting to an external job management system.

Reply:
Personally I'd prefer to stick with "background", at least for the moment. In the future it won't really matter since users will only need to worry about platforms and you should never need to specify "job runner = background" in the global config.

Reply:
OK, let's leave it for now.